### PR TITLE
scikit-image: init at 0.11.3

### PR DESCRIPTION
**Pull request by datakurre**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -15707,6 +15709,26 @@ let
     };
   };
 
+  scikitimage = buildPythonPackage rec {
+    name = "scikit-image-${version}";
+    version = "0.11.3";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/scikit-image/scikit-image-0.11.3.tar.gz";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: You can use string interpolation to avoid repeating the name.

Comment (datakurre) [inline pkgs/top-level/python-packages.nix]: Thanks. Forgot that.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -8626,6 +8626,8 @@ let
       ${self.python.executable} runtests.py
     '';
 
+    __impureHostDeps = optionals stdenv.isDarwin [ "/usr/lib/libm.dylib" ];
+
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: is this to make numba work?

Comment (datakurre) [inline pkgs/top-level/python-packages.nix]: it's to get llvmlite build, it was a broken (on darwin) dependency for scikit-image; numba is still broken on darwin

**Comment by vcunat** [comment]
I believe it would be better to use dashes to unify regular and attribute names. I haven't changed anything (yet), as we have `scikitlearn` in there already...

**Comment by FRidh** [comment]
@vcunat we actually discussed the naming a bit here at the sprint. Personally, I prefer to use lowercase everywhere and remove dashes and underscores. Others feel more for keeping the name exactly as how it is on PyPI, for several reasons.

**Comment by vcunat** [comment]
Oh, I've missed the discussion then, unfortunately. We have this documented actually for general case http://nixos.org/nixpkgs/manual/#sec-package-naming. There might be good reasons for python stuff to have different guidelines, like compatibility with PyPI, I don't know (not really using python myself).

**Pull request closed without merging**